reference/api/settings.mdx

```diff
@@ -1910,3 +1911,189 @@ Reset an index's typo tolerance settings to their [default value](#typo-toleranc
+| **`url`**              | String           | `http://localhost:11434/api/embeddings`                                  | URL Meilisearch contacts when querying the embedder                                                                                                        |
+| **`apiKey`**           | String           | Empty                                                                    | Authentication token Meilisearch should send with each request to the embedder. If not present, Meilisearch will attempt to read it from environment variables |
+| **`model`**            | String           | Empty                                                                    | The model your embedder uses when generating vectors                                                                                                           |
+| **`documentTemplate`** | String           | `{% for field in fields %}{{field.name}}: {{field.value}}\n{% endfor %}` | Template defining the data Meilisearch sends the embedder                                                                                                      |
+| **`dimensions`**       | Integer          | Empty                                                                    | Number of dimensions in the chosen model. If not supplied, Meilisearch tries to infer this value                                                               |
+| **`revision`**         | String           | Empty                                                                    | Model revision hash                                                                                                                                            |
+| **`inputField`**       | Array of strings | []                                                                       | Path to location of document data in query Meilisearch sends to embedder                                                                                       |
+| **`inputType`**        | String           | `text`                                                                   | Type of data Meilisearch sends to embedder. Must be `text` or `textArray`                                                                                      |
+| **`query`**            | String           | Object                                                                   | Extra fields Meilisearch adds to queries Meilisearch sends to embedder                                                                                         |
+| **`pathToEmbeddings`** | Array            | []                                                                       | Path to vector embedding data in embedder response                                                                                                             |
```

Comment:
Path to the location of embedding objects in the response. Useful for `inputType = textArray`.

Reply:
Just to be sure I understand this and `embeddingObject` correctly.

If `pathToEmbeddings` is `["A", "B", "C"]`, Meilisearch will look for the embedding objects in `response.A.B.C`, correct?

What happens if someone uses the wrong path? Will we throw an error? Or will the documents' `_vector` field be null/undefined?

Reply:
If `pathToEmbeddings` is `["A", "B", "C"]` and `embeddingObject` is `["D", "E", "F"]`

1. Meilisearch will look for the embedding objects in `response.A.B.C`
  - If any of `response`, `A` or `B` is not an object, then an error will be thrown
2. If `inputType == "text"`, Meilisearch will look for the embedding in `response.A.B.C.D.E.F`.
  - If any of `C`, `D`, `E` is not an object, then an error will be thrown
  - if `F` is not an array of numbers, then an error will be thrown
3. If `inputType == "arrayOfTexts"`, then Meilisearch will handle `response.A.B.C` as an array of elements, and for each element in elements, will look in `element.D.E.F`
  - If `C` is not an array, then an error will be thrown
  - If any of `element`, `D` or `E` is not an object, then an error will be thrown
  - If `F` is not an array of numbers, then an error will be thrown